pkg/network/usm/kafka_monitor_test.go

```diff
@@ -380,23 +409,39 @@ func (i *PrintableInt) String() string {
 	return fmt.Sprintf("%d", *i)
 }
 
+func (i *PrintableInt) Load() int {
```

Comment:
Why we need a PrintableInt?

Reply:
To print the actual value when `require.Eventually` fails

Reply:
Why not using %d?